solr/core/src/java/org/apache/solr/util/StartupLoggingUtils.java

```diff
@@ -151,6 +153,11 @@ public static void shutdown() {
     }
     flushAllLoggers();
     LogManager.shutdown(true);
+
+    // re-instate original log level.
+    if (!INITIAL_ROOT_LOG_LEVEL.equals(getLogLevelString())) {
+      changeLogLevel(INITIAL_ROOT_LOG_LEVEL);
+    }
```

Comment:
The move of this re-instate / reset of the root log level into shutdown() is debatable.  I was trying to reduce little details in STC/STCJ4.